_im-plugin/ism/api.md

```diff
@@ -546,3 +546,150 @@ DELETE _plugins/_ism/policies/policy_1
   "_primary_term": 1
 }
 ```
+
+## Error prevention validation
+Introduced 2.4
+{: .label .label-purple }
+
+ISM allows you to execute an action automatically. However, action execution can fail for a variety of reasons. You can use error prevention validation to dry run an action in order to rule out failures.
```

Comment:
Rather than "execute" can we use "run" or "start"? Let's use more formal wording than "dry run".